test/e2e/funcs/feature.go

```diff
@@ -385,6 +390,72 @@ func DeleteResources(dir, pattern string) features.Func {
 	}
 }
 
+// CopyImageToRegistry tries to copy the supplied image to the supplied registry within the timeout
+func CopyImageToRegistry(clusterName, ns, sName, image string, timeout time.Duration) features.Func {
+	return func(ctx context.Context, t *testing.T, c *envconf.Config) context.Context {
+		reg, err := ServiceIngressEndPoint(ctx, c, clusterName, ns, sName)
```

Comment:
perhaps we should detect if registry is available and if not fail with a meaningful message?